views/account/settings.hbs

```diff
@@ -41,6 +40,9 @@
                     <div class="input-group show_hide_password">
                         <input type="password" autocomplete="false" id="settings_current_password" data-testid="settings_password_current" name="password" class="form-control"
                         required placeholder="***************">
+                        <div class="input-group-addon">
+                            <button type="button" title="{{$t "login.button.show_password"}}" style="cursor: pointer;"><i class="fa fa-eye-slash" aria-hidden="true"></i></button>
```

Comment:
I know that it is quite common to use the `<i>` tag for icons. But I am not sure if we should also use it this way or we should switch to (the semantically more correct) `<span>` tag.